crossing-training.lic

```diff
@@ -35,6 +35,7 @@ class CrossingTraining
     @swimming_target = get_data('athletics').swimming_options[@settings.swimming_target]
     @song_list = get_data('perform').perform_options
 
+    @attunement_timer = Time.now - @settings.attunement_interval
```

Comment:
From my understanding of the feature request, this would be in addition to the power walking training that is normally done by crossing-training. Adding the 60 second timer to the normal experience timers would remove the ability to maintain attunement training while delaying the frequency of power walking.